src/cordova/setup.action.mjs

```diff
@@ -62,6 +62,12 @@ export async function main(...parameters) {
   if (isApple) {
     // since apple can only be build on darwin systems, we don't have to worry about windows support here
     // TODO(daniellacosse): move this to a cordova hook
+    execSync(`cd third_party/CocoaLumberjack && make clean && make`, {
```

Comment:
It's a bit of a pain to track these dependencies. I'm not sure this is the best place to do that.
Perhaps we add the make command to the post install script of the plugin package.json?
Perhaps we make this a package itself, with a make step and depend on it from the plugin package.json?

Reply:
Another possibility is as a plugin hook.

Reply:
There is a TODO here already for making this part a hook, i was rolling this into that